Source/FakeItEasy.Tests/Creation/DefaultFakeCreatorTests.cs

```diff
@@ -114,6 +121,21 @@ public void CreateFake_should_pass_options_builder_that_returns_itself_for_any_c
             Assert.That(call.Invoke(builderPassedToAction), Is.SameAs(builderPassedToAction));
         }
 
+        [TestCaseSource("invalidOptionBuilderCalls")]
+        public void CreateFake_should_throw_when_type_passed_to_options_builder_is_not_an_interface(Func<IFakeOptionsBuilder<Foo>, IFakeOptionsBuilder<Foo>> call)

```

Comment:
I went with the convention that existed for the valid instantiations in the test above (CreateFake_should_pass_options_builder_that_returns_itself_for_any_call).  I'm happy to split this out, but it seemed like a decent candidate for parameterising given that there are 3 inputs which should all caused failures now.